.gitignore

```diff
@@ -28,3 +28,7 @@ junit.xml
 *.tsbuildinfo
 
 yarn.lock
+
+# xeus-python debug logs
```

Comment:
Is this intentional?

Reply:
Yes. `xeus-python` needs this folder (for now), and the logs shouldn't be checked in.

Reply:
But isn't that a comment?

Reply:
To give a hint for why we have these log files